packages/cli/src/utils/get-config.ts

```diff
@@ -93,10 +108,7 @@ export async function ensureEnvFile(envFilePath: string) {
 export async function setupPgLite(elizaDbDir: string, envFilePath: string): Promise<void> {
   try {
     // Ensure the PGLite database directory exists
-    if (!existsSync(elizaDbDir)) {
-      await fs.mkdir(elizaDbDir, { recursive: true });
-      logger.info(`Created PGLite database directory: ${elizaDbDir}`);
-    }
+    await ensureProjectPGLiteDir();
```

Comment:
In ensureProjectPGLiteDir(), we're calling getElizaDirectories() again even though elizaDbDir is already available in the calling context
Would it make sense to refactor ensureProjectPGLiteDir() to accept elizaDbDir as a parameter instead?